packages/gatsby-theme-carbon/src/components/AutolinkHeader/AutolinkHeader.module.scss

```diff
@@ -0,0 +1,25 @@
+@import '~carbon-components/scss/globals/scss/vars';
```

Comment:
Only importing the necessary sass. It's important this sass is variables only so it doesn’t generate css on every module.